Dockerfile

```diff
@@ -75,9 +76,10 @@ FROM base-$TARGETARCH$TARGETVARIANT AS base
 
 FROM base AS build
 ARG TARGETPLATFORM
-ARG QEMU_VERSION
+ARG QEMU_REF
+ARG QEMU_REPO
 RUN --mount=target=.,from=src,src=/src/qemu,rw --mount=target=./install-scripts,src=scripts \
-  TARGETPLATFORM=${TARGETPLATFORM} configure_qemu.sh && \
+  TARGETPLATFORM=${TARGETPLATFORM} QEMU_VERSION=v$(curl -L ${QEMU_REPO}/raw/${QEMU_REF}/VERSION) configure_qemu.sh && \
```

Comment:
why github specific curl and not just open local file?

Reply:
right in this case we can use local file